src/neurostatslib/utils.py

```diff
@@ -383,3 +383,34 @@ def row_wise_kron(a, c):
         K = K.T
 
     return K
+
+
+def has_local_device(device_type: str) -> bool:
```

Comment:
does `jax` not have something like `torch.cuda.is_available()`? weird.

Reply:
I don't think they sorted it out yet:

[this](https://github.com/google/jax/issues/971) closed issue seems the more relevant. It mention torch, i did not find a public api method either